src/engine/engine_common.c

```diff
@@ -24,7 +24,7 @@ void ocf_engine_error(struct ocf_request *req,
 	if (stop_cache)
 		env_bit_clear(ocf_cache_state_running, &cache->cache_state);
 
-	ocf_core_log(&cache->core[req->core_id], log_err,
+	ocf_core_log(cache->core, log_err,
```

Comment:
The first argument should be "req->core"

Reply:
Done!